src/lib/logging.ts

```diff
@@ -0,0 +1,69 @@
+import Transport from "winston-transport";
+import { createDefaultTransportFormat } from "@zwave-js/core";
+import type { ZWaveLogInfo } from "@zwave-js/core";
+import { ClientsController, Logger } from "./server";
+import { Driver } from "zwave-js";
+
+export class LoggingEventForwarder {
+  /**
+   * Only load this once the driver is ready.
+   *
+   * @param clients
+   * @param driver
+   */
+  private serverTransport?: WebSocketLogTransport;
+  public started: boolean = false;
+
+  constructor(private clients: ClientsController, private driver: Driver) {
+    // Create log transport for server
+    this.serverTransport = new WebSocketLogTransport(this.clients);
```

Comment:
You could simplify this class by only initializing when `start` is called and then storing the transport as an instance variable. `started` can then be implemented as `this.serverTransport !== undefined`

Reply:
I would have to then delete the instance when I stop as well right? I thought it would make more sense to keep the transport in memory instead of recreating it every time, but I am fine with either approach